budget-binder-common/src/commonMain/kotlin/de/hsfl/budgetBinder/common/Category.kt

```diff
@@ -10,9 +10,53 @@ data class Category(
     val image: Image,
     val budget: Float
 ) {
-    enum class Image {
+    enum class Image { //Should be regrouped, sorted
         DEFAULT,
-        SHOPPING
+        CHECKMARK,
+        WRONG,
+        SHOPPINGCART,
+        SHOPPINGBASKET,
+        FOOD,
+        FASTFOOD,
+        RESTAURANT,
+        MONEY,
+        HOME,
+        FAMILY,
+        HEALTH,
+        MEDICATION,
+        KEYBOARD,
+        PRINTER,
+        INVEST,
+        SPORT,
+        CLOTH,
+        GIFT,
+        WEALTH,
+        FLOWER,
+        PET,
+        BILLS,
+        WATER,
+        FIRE,
+        STAR,
+        SAVINGS,
+        CAR,
+        BIKE,
+        TRAIN,
+        MOTORCYCLE,
+        MOPED,
+        ELECTRONICS,
+        BOOK,
+        FLIGHT,
+        WORK,
+        MOON,
+        LOCK,
+        PHONE,
+        STORE,
+        BAR,
+        FOREST,
+        HARDWARE,
+        PEST
+
+
     }
```

Comment:
DEFAULT bleibt oben, muss für 1.0 inital sortiert werden und darf dann nicht mehr umsortiert werden, in der DB steht nämlich nur die Zahl der Position.

Reply:
Im Großen und Ganzen ist die Sortierung sogar egal, da wir im Frontend die Auswahloptionen eh gruppieren können, wie wir wollen.